api-test.json

```diff
@@ -50619,44 +50581,63 @@
 																"disabled": true
 															}
 														],
+														"body": {
+															"mode": "raw",
+															"raw": "",
+															"options": {
+																"raw": {
+																	"language": "json"
+																}
+															}
+														},
 														"url": {
-															"raw": "{{registry-manager}}/ngsi-ld/v1/csourceRegistrations/urn:ngsi-ld:ContextSourceRegistration:cccc",
+															"raw": "{{entity-manager}}/ngsi-ld/v1/entities/urn:test:testentity12345789",
 															"host": [
-																"{{registry-manager}}"
+																"{{entity-manager}}"
 															],
 															"path": [
 																"ngsi-ld",
 																"v1",
-																"csourceRegistrations",
-																"urn:ngsi-ld:ContextSourceRegistration:cccc"
+																"entities",
+																"urn:test:testentity12345789"
 															]
 														}
 													},
 													"response": []
-												}
-											]
-										},
-										{
-											"name": "Attempt update based on exclusive reg",
-											"item": [
+												},
 												{
-													"name": "Create Exclusive Registry",
+													"name": "Create partial matching reg simple Entity check double fail",
 													"event": [
 														{
 															"listen": "test",
 															"script": {
 																"exec": [
 																	"var payloadBody = JSON.parse(pm.request.body.raw);\r",
 																	"console.log(payloadBody.id);\r",
-																	"var cSourceId = payloadBody.id;\r",
+																	"var entityId = payloadBody.id;\r",
 																	"pm.collectionVariables.set(\"created\", payloadBody);\r",
-																	"pm.collectionVariables.set(\"cSourceId\", cSourceId);\r",
-																	"pm.test(\"Status code is 201\", function () {\r",
-																	"    pm.response.to.have.status(201);\r",
+																	"pm.collectionVariables.set(\"entityId\", entityId);\r",
+																	"pm.test(\"Status code is 207\", function () {\r",
```

Comment:
Validate test cases for query in all query parameter.

Reply:
done